internal/handler/handler_test.go

```diff
@@ -558,11 +629,35 @@ func Test_userManagementServiceLogic_User(t *testing.T) {
 				svc := &userMgmtSvc{
 					logic: mockLogic,
 				}
-				by, err := json.Marshal(map[string]interface{}{"user_id": "123"})
+				r := httptest.NewRequest("PUT", "/activate", nil)
+				ctx := session.SetSession(r.Context(), "1234")
+				return svc, r.WithContext(ctx)
+			},
+			want: func(rec httptest.ResponseRecorder) {
+				b, err := ioutil.ReadAll(rec.Body)
 				if err != nil {
-					t.Fail()
+					return
 				}
-				r := httptest.NewRequest("PUT", "/activate", bytes.NewBuffer(by))
+				var response respModel.Response
+				err = json.Unmarshal(b, &response)
+				tempResp := &respModel.Response{
+					Status:  http.StatusOK,
+					Message: codes.GetErr(codes.Success),
+					Data:    nil,
+				}
+				if !reflect.DeepEqual(&response, tempResp) {
+					t.Errorf("Want: %v, Got: %v", tempResp, &response)
+				}
+			},
+		},
+		{
+			name: "Failure",
```

Comment:
One more failure case when assertion to string fails.

Reply:
done